docusaurus/docs/Android/03-ui/06-guides/08-adding-custom-attachments.mdx

```diff
@@ -0,0 +1,169 @@
+# Adding Custom Attachments
+
+## Introduction
+
+By default Stream Chat supports several built-in attachment types like image, video, file and giphy. You can also add your own types of attachments such as location, contact, audio, sticker, etc.
+
+In this guide, we'll demonstrate how to build a date sharing feature. Chat users will be able to pick a date from a calendar, preview their selection in the message input and see the sent attachment within a message in the message list.
+
+This involves doing the following steps:
```

Comment:
Just a proposal, not a mandatory change 🙂
`The following steps are supposed to be made to achieve the goal above:`

Reply:
I think the general guideline we follow with writing docs is to use as few words as possible, even if it means we ditch certain formalities.

It took a while for me to get used to it well, but it works because people inherently lose interest when they are faced with more text.

Reply:
Yeah the shorter and more direct the better. @kanat @MarinTolic @TGazica @bychkovdmitry 

People don't like reading much and want to get quick information. Here are a few examples:

"You can use this new feature to be able to build your own custom attachment factories." This explains things, but is very verbose. Instead, say:

"Use this new feature to build your custom attachment factories."

Much more straightforward, clear and concise, while conveying the same information.